install_config/persistent_storage/topics/glusterfs_storageclass.adoc

```diff
@@ -0,0 +1,67 @@
+.glusterfs-storageclass.yaml
+[source,yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: slow
+provisioner: kubernetes.io/glusterfs
+parameters:
+  resturl: "http://127.0.0.1:8081" <1>
+  restuser: "admin" <2>
+  secretName: "heketi-secret" <3>
+  secretNamespace: "default" <4>
+  gidMin: "40000" <5>
+  gidMax: "50000" <6>
+----
+<1> link:https://github.com/heketi/heketi[heketi] (volume management REST
+service for Gluster) URL that provisions GlusterFS volumes on demand. The
+general format should be `{http/https}://{IPaddress}:{Port}`. This is a
+mandatory parameter for the GlusterFS dynamic provisioner. If the heketi
+service is exposed as a routable service in the {product-title}, it will have a
+resolvable fully qualified domain name and heketi service URL.
+<2> heketi user who has access to create volumes. Usually "admin".
+<3> Identification of a Secret that contains a user password to use when
```

Comment:
Does "Secret" need to be capitalized?

Reply:
It's a resource signifier, so I believe so.